net/lp_rpc.proto

```diff
@@ -82,7 +82,7 @@ message OrchestratorInfo {
   // URI of the transcoder to use for submitting segments.
   string transcoder  = 1;
 
-  // Required parameters for probabilistic micropayment tickets
+  // Parameters for probabilistic micropayment tickets
```

Comment:
Removed the reference to "required" here since it's a bit ambiguous:

* Offchain mode won't have parameters defined
* All fields are optional in protocol buffers anyway